client/data/marketplace/use-es-query.ts

```diff
@@ -54,9 +55,9 @@ const mapIndexResultsToPluginData = ( results: ESHits ): Plugin[] => {
 			support_threads_resolved: hit[ 'plugin.support_threads_resolved' ],
 			active_installs: hit.plugin?.active_installs,
 			last_updated: hit.modified,
-			short_description: hit.plugin?.excerpt, // TODO: add localization
+			short_description: decodeEntities( hit.plugin?.excerpt ), // TODO: add localization
 			icon: getIconUrl( hit.slug, hit.plugin?.icons ),
-			premium_slug: hit.plugin?.premium_slug,
+			premium_slug: decodeEntities( hit.plugin?.premium_slug ),
```

Comment:
Same as above.

Reply:
Similar answer to the above https://github.com/Automattic/wp-calypso/pull/73935#discussion_r1122037313